hotspot/test/testlibrary/whitebox/sun/hotspot/WhiteBox.java

```diff
@@ -256,5 +256,8 @@ public int getOffsetForName(String name) throws Exception {
   // Container testing
   public native boolean isContainerized();
   public native void printOsInfo();
+  public native int validateCgroup(String procCgroups,
```

Comment:
This doesn't look right. It's part of #135. See: https://github.com/openjdk/jdk8u-dev/pull/135/files#diff-840578f66556c33204f36e83d8c5a6b6b91c0280ffdbbea6121847a5b6b1afe7R259